Makefile

```diff
@@ -1,29 +1,17 @@
-.PHONY: all test push build release clean
```

Comment:
Do we need the makefile? For other plugins we dropped the makefile entirely.

Reply:
I would like to keep the `Makefile` for now as I'm using it for local image building and compiling.